lecture_2/homework/app/controllers/application_controller.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 class ApplicationController < ActionController::API
   rescue_from ActiveRecord::RecordNotFound do |exception|
```

Comment:
Z czasem, jak aplikacja będzie rosła to w `ApplicationController` będzie pojawiać się coraz więcej "pomocnicznego" kodu. Żeby zachować w tym porządek polecam zwinąć obsługę błędów do oddzielnego modułu